src/TicketMap/index.test.tsx

```diff
@@ -0,0 +1,355 @@
+import React from 'react'
+import { shallow, ShallowWrapper } from 'enzyme'
+
+import TicketMap from './index'
+import { State, Props, Manifest } from './types'
+import Tooltip from '../Tooltip'
+import Actions from '../Actions'
+import { TicketGroup } from '../types'
+
+describe('TicketMap', () => {
+  let wrapper: ShallowWrapper<Props, State, TicketMap>
+  let props: Props
+  let manifestResponse: [string, { status: number }]
+  let mapResponse: [string, { status: number }]
+  let mapRoot: HTMLDivElement
+  let manifest: Manifest
+  let ticketGroup: TicketGroup
+  let fillSectionSpy: jest.SpyInstance
+
+  const mountComponent = async (responses = [mapResponse, manifestResponse]) => {
+    fetchMock.mockResponses(...responses)
+    wrapper = shallow(<TicketMap {...props} />)
+    wrapper.instance().mapRoot = { current: mapRoot }
+    fillSectionSpy = jest.spyOn(wrapper.instance(), 'fillSection')
+    await new Promise(setImmediate)
+  }
```

Comment:
You can just return this promise in the `beforeEach` callback. Tests will start when the promise resolves.

Check out the second paragraph of [Jest Setup and Teardown: Repeating Setup For Many Tests](https://jestjs.io/docs/en/setup-teardown#repeating-setup-for-many-tests) and the [API doc entry for `beforeEach`](https://jestjs.io/docs/en/api#beforeeachfn-timeout) a bit more detail on this pattern.